docs/DeveloperGuide.adoc

```diff
@@ -272,111 +509,402 @@ Refer to the guide <<Testing#, here>>.
 
 == Dev Ops
 
-Refer to the guide <<DevOps#, here>>.
+=== Build Automation
+
+We use Gradle for build automation. See <<UsingGradle#, here>> for more details.
+
+=== Continuous Integration
+
+We use https://travis-ci.org/[Travis CI] to perform _Continuous Integration_ on our project. See <<UsingTravis#, here>> for more details.
+
+=== Documentation Previews
+
+We use https://www.netlify.com/[Netlify] to preview the HTML pages of any modified asciidocs files when reviewing pull requests. See <<UsingNetlify#, here>> for more details.
+
+=== Making a Release
+
+Follow the steps below to make a new release:
+
+.  Update the version number in link:{repoURL}/src/main/java/io/xpire/MainApp.java[`MainApp.java`].
+.  Generate a JAR file <<UsingGradle#creating-the-jar-file, using Gradle>>.
+.  Tag the repo with the version number. e.g. `v1.4`
+.  https://help.github.com/articles/creating-releases/[Create a new release using GitHub] and upload the JAR file you have created.
```

Comment:
## Kudos!
### Comments
This is very informative and really helped me to understand a better way to release the JAR file.
### Reasoning
I really liked how informative the information is.